projects/interlay-lending/api.js

```diff
@@ -0,0 +1,42 @@
+const { getAPI, addTokenBalance } = require('../helper/acala/interlay-api');
+
+// for exchange rate decoding
+const FIXEDI128_SCALING_FACTOR = 18;
+
+async function tvl() {
+  const chain = "interlay";
+  const api = await getAPI(chain);
+  const balances = {};
+  
+  const data = (await api.query.loans.markets.entries());
+
+  for (let i = 0; i < data.length; i++) {
+    const [underlyingId, marketData] = data[i];
+
+    const isActive = marketData?.toJSON().state == "Active";
+    if (!isActive) {
+      continue;
+    }
+
+    const lendTokenId = marketData.toJSON().lendTokenId;
+
+    const [exchangeRate, totalIssuance] = Promise.all([
```

Comment:
you need to separate borrowed from money thats sitting on the contracts

Reply:
> you need to separate borrowed from money thats sitting on the contracts

Thanks for the feedback.
Just to make sure I understand correctly: Is the requirement to show `locked amounts in lending protocol - current borrowed amounts` ? (ie. only the amounts available for borrowing)

Reply:
yes tvl should = total supplied - borrowed and borrowed can be exported separately just as borrowed

Reply:
> yes tvl should = total supplied - borrowed and borrowed can be exported separately just as borrowed

Thank you for the confirmation. 
Commit [11db388](https://github.com/DefiLlama/DefiLlama-Adapters/pull/8552/commits/11db388b20d0583bfee6ea44330b7c5db2c64fa0) should fix that.